lib/core/timer.cpp

```diff
@@ -6,12 +6,26 @@ Timer::Timer(const ITimeSource &time) : time_(time)
 {
 }
 
-Duration Timer::measure_execution_time(const std::function<void(void)> task)
+Duration Timer::measureExecutionTime(const std::function<void(void)> task)
 {
   const auto before = time_.now();
   task();
   const auto after = time_.now();
   return after - before;
 }
 
+Duration Timer::measureElapsedTime(const TimePoint current_timepoint,
```

Comment:
I don't like this function; I think it should be 

```cpp
Duration Timer::elapsed() { /* ... */ }
```

Reply:
basically, just copy the files from [here](https://github.com/Hyp-ed/hyped-2023/blob/navigation/navigator-benchmarker/lib/core/timer.cpp)